### PR TITLE
[RHACS] Added release note about code_bpf support for ppc64le arch

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on Slack convo:
> I need to get a change added in upcoming 4.4 release doc that says Core-bpf is now supported on ppc64le. Who can help me with that?
> ...
> So I guess mentioning about core-bpf support on Power in just release notes should be fine. Like in this [4.2](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/4.2/html/release_notes/release-notes-42#the-core-bpf-collection-method-is-now-ga_release-notes-42) release notes.

Adds an additional item to the release note.

Preview: https://74023--docspreview.netlify.app/openshift-acs/latest/release_notes/44-release-notes#core-bpf-collection-method-is-now-generally-available_release-notes-44

Cherrypick to `rhacs-docs-4.4`

### CODE REVIEW

File: release_notes/44-release-notes.adoc [hunk 1]
```diff
@@ -182,6 +182,11 @@ The following are some of the advantages of this new collection method:
 * You no longer need to update the Collector support packages if you are offline.
 By avoiding issues related to eBPF probe retrieval, such as the risk of losing the network connection or if the probe is not present, you can update a cluster smoothly.
 
+[id="core-bpf-collection-method-is-now-generally-available_{context}"]
+=== The CORE BPF collection method is now generally available for ppc64le architecture
+
+{product-title-short} 4.2 included the runtime collection method based on BPF CO-RE (Compile Once-Run Everywhere), which is available on the `x86_64` and `s390x` architectures. Starting with {product-title-short} 4.4, BPF CO-RE is now generally available on the `ppc64le` architecture. To enable it, set the value of your backed-up cluster's `collector.collectionMethod` parameter to `CORE_BPF`.
```
Comment (msugakov) [inline release_notes/44-release-notes.adoc:188]:
```suggestion
{product-title-short} 4.2 included the runtime collection method based on BPF CO-RE (Compile Once-Run Everywhere), which is available on the `x86_64` and `s390x` architectures. Starting with {product-title-short} 4.4, BPF CO-RE is now generally available on the `ppc64le` architecture. To enable it, set the value of your secured cluster's `collector.collectionMethod` parameter to `CORE_BPF`.
```

**Review by msugakov — APPROVED** [review]

**Comment by nalhadef** [comment]
No comments. Looks good!

**Pull request merged**